src/index.js

```diff
@@ -130,9 +132,10 @@ export default function onClickOutsideHOC(WrappedComponent, config) {
           events = [events];
         }
         events.forEach(eventName => {
-          const handlerOptions = !this.props.preventDefault && ['touchstart', 'touchmove'].indexOf(eventName) !== -1
-            ? { passive: true }
-            : null;
+          const handlerOptions =
```

Comment:
can you also update the `handlerOptions` though? it's much cleaner as code that always sets up an object.